src/core/core.scale.js

```diff
@@ -857,15 +857,11 @@ module.exports = function(Chart) {
 
 					var label = itemToDraw.label;
 					if (helpers.isArray(label)) {
-						var lineCount = label.length;
```

Comment:
I think you're accidentally reverting a recently made change: https://github.com/chartjs/Chart.js/commit/584d1c646c84f3d49c865ab69e774cc6541027a4

It looks like this file doesn't need to be on the pull request

Reply:
yep, thats file is not be here, i will fix this